simularium_metrics_calculator/metrics_registry.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from typing import Dict
+
+from .calculators import (
+    AgentsCalculator,
+    NearestNeighborCalculator,
+    NumberOfAgentsCalculator,
+    TimesCalculator,
+)
+from .constants import METRIC_TYPE
+from .exceptions import MetricNotFoundError
+from .metric_info import MetricInfo
+
+metrics_list = [
+    MetricInfo(
+        uid=0,
+        display_name="Time",
+        metric_type=METRIC_TYPE.PER_TIME,
+        calculator=TimesCalculator,
+    ),
+    MetricInfo(
+        uid=1,
+        display_name="Agent IDs",
+        metric_type=METRIC_TYPE.PER_AGENT,
+        calculator=AgentsCalculator,
+    ),
+    MetricInfo(
+        uid=2,
+        display_name="Number of agents",
+        metric_type=METRIC_TYPE.PER_TIME,
+        calculator=NumberOfAgentsCalculator,
+    ),
+    MetricInfo(
+        uid=3,
```

Comment:
one question about the uids:  do you expect them to ever be saved anywhere or need to persist across sessions?  That decision would affect whether you can dynamically generate them or need to hardcode them.  If they will need to be persisted then the uid becomes a "permanent id".

Reply:
I can't think of any reason they need to persist across sessions, so now they are dynamically generated from indices of the metrics list.